spring-cloud-deployer-local/src/main/java/org/springframework/cloud/deployer/spi/local/AbstractLocalDeployerSupport.java

```diff
@@ -186,7 +186,8 @@ protected ProcessBuilder buildProcessBuilder(AppDeploymentRequest request, Map<S
 
 			if(request.getResource() instanceof DockerResource) {
 				builder.command().add(2, "-e");
-				builder.command().add(3, "JAVA_TOOL_OPTIONS=\""+ debugInstruction + "\"");			}
+				builder.command().add(3, "JAVA_TOOL_OPTIONS="+ debugInstruction);
```

Comment:
This is not adding support for passing the deployer property `spring.cloud.deployer.local.javaOpts`.  It appears to be fixing something wrt to debug support, which afaik, we had tests for that were working.